### PR TITLE
libldac: refactoring debian/

### PR DESCRIPTION
**Pull request by souravdas142**
see commit message

Is that fine ? @tormodvolden

### CODE REVIEW

File: libldac/debian/copyright [hunk 1]
```diff
@@ -10,7 +10,8 @@ Copyright: 2003-2017, Sony Corporation
 License: Apache-2
 
 Files: debian/*
-Copyright: 2020 Sourav Das <souravdas142@gmail.com>
+Copyright: 2020 Andrej Shadura <andrewsh@debian.org>,
+           2022 Sourav Das <souravdas142@gmail.com>
```
Comment (tormodvolden) [inline libldac/debian/copyright:14]: You don't need the comma at the end, this field is free format (https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/#copyright-field).

**Comment by tormodvolden** [comment]
Yes, this looks good: The Replace/Conflicts against the same binary package didn't make sense. One version of a package cannot be installed at the same time as another version of the same package, so it cannot conflict. And a package always replaces the files from the same package.

If there really are conflicts between the different binary packages of this source package, you should declare them, but I don't see any reason here.

**Comment by pipewire-debian** [comment]
> Yes, this looks good: The Replace/Conflicts against the same binary package didn't make sense. One version of a package cannot be installed at the same time as another version of the same package, so it cannot conflict. And a package always replaces the files from the same package.
> 
> If there really are conflicts between the different binary packages of this source package, you should declare them, but I don't see any reason here.

Yeah, True. Anything else I should change ?

**Comment by tormodvolden** [comment]
I think this is good.

**Pull request merged**